benchmarks/gaussian_localisation/gaussian_test.py

```diff
@@ -0,0 +1,137 @@
+import argparse
+import dask
+import glob
+import re
+import sys
+import os
+
+import deepblink as pink
+import numpy as np
+import scipy.optimize as opt
+
+sys.path.append("../")
+from util import delayed_results
+
+EPS = 1e-4
+
+
+def _parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--datasets")
+    parser.add_argument("--models")
+    args = parser.parse_args()
+    return args
+
+
+@dask.delayed
+def delayed_normalize_pink(image):
+    return pink.data.normalize_image(image)
+
+
+def gauss_2d(xy, amplitude, x0, y0, sigma_xy, offset):
+    """2D gaussian."""
+    x, y = xy
+    x0 = float(x0)
+    y0 = float(y0)
+    gauss = offset + amplitude * np.exp(
+        -(
+            ((x - x0) ** (2) / (2 * sigma_xy ** (2)))
+            + ((y - y0) ** (2) / (2 * sigma_xy ** (2)))
+        )
+    )
+    return gauss
+
+
+def gauss_single_spot(
+    image: np.ndarray, c_coord: float, r_coord: float, crop_size: int
+):
+    """Gaussian prediction on a single crop centred on spot."""
+    start_dim1 = np.max([int(np.round(r_coord - crop_size // 2)), 0])
+    if start_dim1 < len(image) - crop_size:
+        end_dim1 = start_dim1 + crop_size
+    else:
+        start_dim1 = len(image) - crop_size
+        end_dim1 = len(image)
+
+    start_dim2 = np.max([int(np.round(c_coord - crop_size // 2)), 0])
+    if start_dim2 < len(image) - crop_size:
+        end_dim2 = start_dim2 + crop_size
+    else:
+        start_dim2 = len(image) - crop_size
+        end_dim2 = len(image)
+
+    crop = image[start_dim1:end_dim1, start_dim2:end_dim2]
+
+    x = np.arange(0, crop.shape[1], 1)
+    y = np.arange(0, crop.shape[0], 1)
+    xx, yy = np.meshgrid(x, y)
+
+    # Guess intial parameters
+    x0 = int(crop.shape[0] // 2)  # Middle of the crop
+    y0 = int(crop.shape[1] // 2)  # Middle of the crop
+    sigma = max(*crop.shape) * 0.1  # 10% of the crop
+    amplitude_max = np.max(crop) / 2  # Maximum value of the crop
+    initial_guess = [amplitude_max, x0, y0, sigma, 0]
+
+    lower = [np.min(crop), 0, 0, 0, np.min(crop)]
```

Comment:
Maybe add one short comment what lower, upper is. And what the components are. I won't touch the code later but still good to know if someone would look through.